string robot_name #Must be: 'robot1' or 'robot2'
string box_name
bool attach #True to attach, False to release

---------------
string message
bool success